packages/sdk-middleware-http/src/http.js

```diff
@@ -91,9 +98,9 @@ export default function createHttpMiddleware({
                   ...requestObj,
                   headers: parseHeaders(requestObj.headers),
                 }
-                if (maskSensitiveHeaderData)
-                  parsedResponse.request.headers.authorization =
-                    'Bearer ********'
+                if (maskSensitiveHeaderData) {
```

Comment:
Hi @daern91 .. what do you think about moving this condition to `maskAuthData` function? So we will test if it is enabled only on one place.

Reply:
Great idea, thanks, done!